core/templates/pages/topic-editor-page/subtopic-editor/subtopic-editor-tab.component.css

```
@@ -0,0 +1,349 @@
+  justify-content: space-between;
+  padding: 10px 0;
+  position: relative;
+}
+.skill-list-item a {
+  color: inherit;
+  flex: 1;
+  overflow: hidden;
+  padding-left: 5px;
+  padding-right: 10px;
+  text-decoration: none;
+  word-wrap: break-word;
```

Stylelint (17.13.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)